### PR TITLE
profiles/package.mask: mask dev-haskell/pandoc-citeproc for removal

### PR DESCRIPTION
**Pull request by ezzieyguywuf**
This package has been deprecated upstream, and will block newer pandoc
if we keep it around.

Signed-off-by: Wolfgang E. Sanyer <ezzieyguywuf@gmail.com>

### CODE REVIEW

**Comment by ezzieyguywuf** [comment]
@trofi ping, as discussed on IRC.

File: profiles/package.mask [hunk 1]
```diff
@@ -32,6 +32,11 @@
 
 #--- END OF EXAMPLES ---
 
+# Wolfgang E. Sanyer <WolfgangESanyer@Gmail.com> (2021-03-11)
+# Per upstream, this has been deprecated in favour if citeproc. bug #775545
+# opened per the examples - Masked for removal in 30 days
+dev-haskell/pandas-citeproc
```
Comment (trofi) [inline profiles/package.mask:38]: I think you spelled it wrong.

Comment (ezzieyguywuf) [reply profiles/package.mask]: Fixed - I was thinking of the python `pandas` library for some reason, I have no idea why :-P

File: profiles/package.mask [hunk 2]
```diff
@@ -32,6 +32,12 @@
 
 #--- END OF EXAMPLES ---
 
+# Wolfgang E. Sanyer <WolfgangESanyer@Gmail.com> (2021-03-11)
```
Comment (trofi) [inline profiles/package.mask:35]: The mask claims it masks deprecated package. But it also masks last available version of softwware maintained by someone else. You will need to sort out the plan with them first.

**Pull request closed without merging**